velox/dwio/common/Statistics.h

```diff
@@ -380,6 +380,69 @@ class IntegerColumnStatistics : public virtual ColumnStatistics {
   std::optional<int64_t> sum_;
 };
 
+// Statistics for decimal columns. T could be int64_t or int128_t.
+template <typename T>
+class DecimalColumnStatistics : public virtual ColumnStatistics {
```

Comment:
Template IntegerColumnStatistics similar to filter?

Reply:
Thanks for the suggestion. Templated  `IntegerColumnStatistics` to support int128_t.